components/common/WalletModal.vue

```diff
@@ -209,6 +220,13 @@ export default class WalletModal extends Vue {
     background: unset;
   }
 
+  .modal-card-body {
+    height: 303px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
Why do you need all this css (fixed height)? and what does it solve?
I think you can fix it in another way, for example with a `block` class

Reply:
so I added a fixed width so that there isn't any flicker once you click on the checkbox and its a smooth transition, without this it would be like this 

https://www.loom.com/share/b5aaa63f029d4f2a8aff383b432a5172